web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -108,8 +130,9 @@ const MemberSelectorDialog = ({
   const [filter, setFilter] = useState(null);
   const [filteredMembers, setFilteredMembers] = useState([]);
   const [directReportsOnly, setDirectReportsOnly] = useState(false);
-
   const [selectableMembers, setSelectableMembers] = useState([]);
+  const [tenure, setTenure] = useState(initialFilter?.tenure || Tenures.All);
```

Comment:
The initial value of the `tenure` state is the one specific in `initialFilter`. But if that isn't present, default to `Tensures.All`.